File: Typescript/Practice/26072022/medium/medium_concateNumbers.py
#!/usr/bin/env python

def concat(*args):
  result = []
  for lst in args:
    result.extend(lst)
  return result

print(concat([1, 2, 3], [4, 5], [6, 7]))
print(concat([1], [2], [3], [4], [5], [6], [7]))
print(concat([1, 2], [3, 4]))
print(concat([4, 4, 4, 4, 4]))

File: Typescript/Practice/medium/medium_concateNumbers.ts
#!/usr/bin/env npx ts-node

function concat(...args: Array<number>[]) {
  let result: number[] = [];
  for (let arr of args) {
    result = result.concat(arr);
  };
  return result;
};


console.log(concat([1, 2, 3], [4, 5], [6, 7]));
console.log(concat([1], [2], [3], [4], [5], [6], [7]));
console.log(concat([1, 2], [3, 4]));
console.log(concat([4, 4, 4, 4, 4]));
